app/workers/assembly_coin/maintain_btc_balance.rb

```diff
@@ -3,55 +3,11 @@ class MaintainBtcBalance < AssemblyCoin::Worker
 
     def perform(product_id)
       product = Product.find_by(id: product_id)
-
       if product.state == 'greenlit' or product.state == 'profitable'
         destination = product.wallet_public_address
-
-        current_btc = OpenAssets::Transactions.new.get_btc_balance(destination)
-        current_time =  Time.now
-
-        last_checked_time = product.last_checked_btc
-        if last_checked_time.nil?
-          last_checked_time = 0
-        end
-
-        time_since_last_checked = current_time.to_i - last_checked_time.to_i
-
-        if ENV['MIN_PRODUCT_BTC'].to_f > current_btc and time_since_last_checked > 3600 then
-          amount = get_send_amount(current_btc)
-          product.update!(last_checked_btc: current_time)
-
-          send_btc_with_logging(product, destination, amount)
-        else
-          Rails.logger.info "[coins] #{product.id} wallet has sufficient Bitcoin"
-        end
+        amount = ENV['MIN_PRODUCT_BTC'].to_f
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.